services/cadip/config/cadip_ws_config.yaml

```diff
@@ -132,96 +129,88 @@ ins:
-        PublicationDate:
-            - null
-            - '$.PublicationDate'
-        EvictionDate:
-            - null
-            - '$.EvictionDate'
-        Size:
-            - null
-            - '$.Size'
-        Retransfer:
-            - null
-            - '$.Retransfer'
-        # Eodag Specific map
-        startTimeFromAscendingNode:
-            - null
-            - '$.PublicationDate'
-        completionTimeFromAscendingNode:
-            - null
-            - '$.PublicationDate'
-        href:
-          - null
-          - 'http://127.0.0.1:5100/Files({id})/$value'
-        geometry: 'POLYGON((180 -90, 180 90, -180 90, -180 -90, 180 -90))'
-        downloadLink: 'http://127.0.0.1:5100/Files({id})/$value'
-        title: '$.Name'
-
+      id:
+      - null
+      - $.Id
+      Name:
+      - null
+      - $.Name
+      SessionIds:
+      - null
+      - $.SessionID
+      SessionID:
+      - null
+      - $.SessionID
+      Channel:
+      - null
+      - $.Channel
+      BlockNumber:
+      - null
+      - $.BlockNumber
+      FinalBlock:
+      - null
+      - $.FinalBlock
+      PublicationDate:
+      - null
+      - $.PublicationDate
+      EvictionDate:
+      - null
+      - $.EvictionDate
+      Size:
+      - null
+      - $.Size
+      Retransfer:
+      - null
+      - $.Retransfer
+      startTimeFromAscendingNode:
+      - null
+      - $.PublicationDate
+      completionTimeFromAscendingNode:
+      - null
+      - $.PublicationDate
+      href:
+      - null
+      - http://127.0.0.1:5100/Files({id})/$value
+      geometry: POLYGON((180 -90, 180 90, -180 90, -180 -90, 180 -90))
+      downloadLink: http://127.0.0.1:5100/Files({id})/$value
+      title: $.Name
     free_text_search_operations:
       $filter:
         union: ' OR '
         wrapper: '"{}"'
         operations:
-            and:
-              - 'SessionID in {SessionIds}'
-              - 'SessionID eq {SessionID}'
-              - 'PublicationDate gt {startTimeFromAscendingNode#to_iso_utc_datetime}'
-              - 'PublicationDate lt {completionTimeFromAscendingNode#to_iso_utc_datetime}'
-
+          and:
+          - SessionID in {SessionIds}
+          - SessionID eq {SessionID}
+          - PublicationDate gt {startTimeFromAscendingNode#to_iso_utc_datetime}
+          - PublicationDate lt {completionTimeFromAscendingNode#to_iso_utc_datetime}
     pagination:
       max_items_per_page: 1000
       next_page_url_tpl: '{url}?{search}&$top={items_per_page}'
-      total_items_nb_key_path: '$.total'
-
+      total_items_nb_key_path: $.total
   download:
     type: HTTPDownload
-    base_uri: 'http://127.0.0.1:5100/Files'
+    base_uri: http://127.0.0.1:5100/Files
     metadata_mapping:
-        uid: '$.Id'
-        downloadLink: 'http://127.0.0.1:5100/Files({uid})/$value'
-
+      uid: $.Id
+      downloadLink: http://127.0.0.1:5100/Files({uid})/$value
 mps:
   auth:
     type: TokenAuth
-    auth_uri: 'http://127.0.0.1:0/oauth2/token'
```

Comment:
Port typo, should be 5101.

Since this file is only applicable in local mode, i'm not sure we really use that information, @agrosu1978  should we update or remove them?

Reply:
In fact the values from this file will be merged into rs-helm and rs-infrastructure so maybe we need the values here

Reply:
It is used by eodag (when it is configured to handle by itself the token from the external stations), i guess it should not be removed